share/wake/lib/core/examples.wake

```diff
@@ -0,0 +1,134 @@
+# Copyright 2021 SiFive, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You should have received a copy of LICENSE.Apache2 along with
+# this software. If not, you may obtain a copy at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+package wake
```

Comment:
Let's not have this in the main package.

Reply:
The examples also don't belong in this directory. They belong in `share/doc/wake/tour/json.adoc` and/or unit tests.

Reply:
Ah I see going through comments backwards has left my later comments somewhat confusing, I'll move this over; I was planning on deleting this at the end of review rather than adding it at all but good to know that it has a home.